src/ziggurat/streams.clj

```diff
@@ -273,12 +291,23 @@
                    stream           (start-stream* topic-handler-fn stream-config topic-entity channels)]
                (when-not (nil? stream)
                  (.start stream)
-                 (conj streams stream))))
-           []
+                 (assoc streams topic-entity stream))))
+           {}
            stream-routes)))
 
+(defn start-stream
+  [topic-entity]
+  (reset! restarted-streams (reduce (fn [new-stream-map stream-object]
+                                      (if (and (= (.state (second stream-object)) org.apache.kafka.streams.KafkaStreams$State/NOT_RUNNING)
+                                               (= (first stream-object) topic-entity))
```

Comment:
`(first stream-object)` is being used at 3 places. Extract this out in a variable inside a let block

Also extract out `(second stream-object)` into a variable. It becomes easier to read and conveys the intent.

Reply:
done